tests/tt_metal/tt_metal/perf_microbenchmark/10_dram_read_remote_cb_sync/test_dram_read_remote_cb.cpp

```diff
@@ -472,17 +476,19 @@ bool validation_mixed_df(
             page_size = 1088;
         }
         layer_transfer_size = page_size * kt * nt / num_receivers;
-        num_pages = fifo_size / page_size;
-        fifo_size_page_aligned = page_size * num_pages;
 
-        bool fifo_wr_ptr_exceed_fifo_limit = fifo_wr_ptr > fifo_size_page_aligned;
-        uint32_t num_pages_till_fifo_limit = (fifo_size_page_aligned - fifo_wr_ptr) / page_size;
+        uint32_t block_size = block_num_tiles * 32*32*2; // fp16
```

Comment:
Could we use defined constants instead of hardcoding?

Ex `TILE_HW * datum_size(DataFormat::Float16_b)`?